From: Anatol Belski <ab@php.net>
Date: Fri, 27 Oct 2017 16:19:42 +0200
Subject: Skip test on PostgreSQL 10

The 42P18 error is not produced by the server anymore.

(cherry picked from commit php/php-src@15a71fe045a476e864900657fc8d4d3586aafd02)
---
 ext/pdo_pgsql/tests/bug48764.phpt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/pdo_pgsql/tests/bug48764.phpt b/ext/pdo_pgsql/tests/bug48764.phpt
index 83fa565..14c1f68 100644
--- a/ext/pdo_pgsql/tests/bug48764.phpt
+++ b/ext/pdo_pgsql/tests/bug48764.phpt
@@ -12,7 +12,7 @@ $db = PDOTest::factory();
 $client_version = $db->getAttribute(PDO::ATTR_CLIENT_VERSION);
 $server_version = $db->getAttribute(PDO::ATTR_SERVER_VERSION);
 
-if (version_compare($server_version, '7.4', '<') || version_compare($client_version, '7.4', '<')) {
+if (version_compare($server_version, '7.4', '<') || version_compare($client_version, '7.4', '<') || version_compare($server_version, '10', '>=')) {
         die('skip');
 }
 
